api/src/main/java/org/openmrs/module/idgen/SequentialIdentifierGenerator.java

```diff
@@ -31,6 +31,7 @@
 	private Integer minLength; // If > 0, will always return identifiers with a minimum of this length
 	private Integer maxLength; // If > 0, will always return identifiers no longer than this length
     private String baseCharacterSet; // Enables configuration in appropriate Base
+	private Integer nextSequenceValue; // Pre-generate the next identifier
 
```

Comment:
I'm pretty sure we deliberately left this off of the class in order to protect it, though if you need access to it for your use case I suppose it doesn't hurt.  Do you find that you need to manually set this "nextSequenceValue" frequently?